src/api/app/models/package.rb

```diff
@@ -234,7 +234,8 @@ def self.get_by_project_and_name!(project, package, opts = {})
 
   # to check existens of a project (local or remote)
   def self.exists_by_project_and_name(project, package, opts = {})
```

Comment:
One more instance of `allow_remote_packages: false` should be removed here:

https://github.com/openSUSE/open-build-service/blob/e9a6fa9c0e7ab5993a0be9d23812b0b6d1e8c84e/src/api/app/models/channel_binary.rb#L39